labrador/src/transcript/sponges/mod.rs

```diff
@@ -1,6 +1,7 @@
 use crate::ring::{rq::Rq, zq::Zq};
 
 pub trait Sponge {
+    fn default() -> Self;
```

Comment:
Make it implement `Default` instead of adding `default()` inside `Sponge`.
```suggestion
pub trait Sponge: Default {
```